src/util/load.js

```diff
@@ -43,33 +42,36 @@ export function events (func, eventName) {
 }
 
 export async function funcs (pattern = '*') {
-  const funcs = readdirSync('functions').filter(minimatch.filter(pattern))
+  const funcs = await readdir('functions').filter(minimatch.filter(pattern))
   if (funcs.length === 0) {
     throw new Error(`No functions found matching patterns: ${JSON.stringify(funcs)}`)
   } else {
     return funcs
   }
 }
 
-export function lambdaConfig (name) {
-  const functionConfig = readJSONSync(`functions/${name}/lambda.json`)
-  const projectConfig = readJSONSync(`lambda.json`)
+export async function lambdaConfig (name) {
+  const functionConfig = await readJSON(`functions/${name}/lambda.json`)
+  const projectConfig = await readJSON(`lambda.json`)
 
   return Object.assign(projectConfig, functionConfig)
 }
 
-export function pkg () {
-  return readJSONSync('package.json')
+export async function pkg () {
+  const pkg = await readJSON('package.json')
+  if (!pkg || !pkg.shep) { throw new Error('Missing shep section in package.json') }
+  return pkg
 }
 
-export function api () {
+export async function api () {
   try {
-    return readJSONSync('api.json')
+    const api = await readJSON('api.json')
+    return api
   } catch (e) {
     return null
   }
 }
 
-export function babelrc () {
-  return readJSONSync('.babelrc')
+export async function babelrc () {
```

Comment:
@reconbot Is it good/bad practice to mark functions that return a promise as `async` even if they don't use `await` in the body?